data/sql/updates/pending_db_world/rev_1621105426491747047.sql

```diff
@@ -0,0 +1,8 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1621105426491747047');
+
+-- Remove id=1: Silvermane Stalker - On Reset - Cast Faded (6408) at 100% chance (No Repeat)
+DELETE FROM `smart_scripts` WHERE `entryorguid`=2926 AND `id`=1;
+
```

Comment:
```  UPDATE `creature_template_addon` SET `bytes1` = 0 WHERE (`entry` = 2926);```  

Add this to resolve missing nameplates

Reply:
Thank you!
Now the nameplate is visible.